guides/plugins/apps/shipping-methods.md

```diff
@@ -0,0 +1,276 @@
+            <description lang="de-DE">Das ist eine einfache Beschreibung</description>
+            <description lang="fr-FR">C'est une description simple</description>
+            ...
+        </shipping-method>
+
+    </shipping-methods>
+</manifest>
+```
+
+### Icon
+You can initially add an icon.
+The path to this icon must be specified as relative to the manifest.xml file. For example, you have the following directory structure:
+```
+- YourPluginDirectory
+    - assets
+        - icons
+            - yourIcon.png
+    - manifest.xml
+```
+The path should be: `assets/icons/yourIcon.png`
+
+::: info
+Please note that the icon cannot be modified by the manifest, as the merchant has the ability to change it.
+:::
+```xml
+// manifest.xml
+
+<?xml version="1.0" encoding="UTF-8" ?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/Manifest/Schema/manifest-2.0.xsd">
+    
+    ...
+
+    <shipping-methods>
+
+        <shipping-method>
+            <identifier>NameOfYourFirstShippingMethod</identifier>
+            <name>First shipping method</name>
+            ...
+            <icon>assets/icons/yourIcon.png</icon>
+            ...
+        </shipping-method>
+
+    </shipping-methods>
+</manifest>
+```
+### Active 
+You can activate the shipping method by default. Possible values for active are `true` or `false`
+
+- true: Activates the shipping method
+- false: The shipping method is deactivated. Alternatively, you can simply leave out active
+
+```xml
+// manifest.xml
+
+<?xml version="1.0" encoding="UTF-8" ?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/Manifest/Schema/manifest-2.0.xsd">
+    
+    ...
+
+    <shipping-methods>
+
+        <shipping-method>
+            <identifier>NameOfYourFirstShippingMethod</identifier>
+            <name>First shipping method</name>
+            ...
+            <active>true</active>
+            ...
+        </shipping-method>
+
+    </shipping-methods>
+</manifest>
+```
+
+### Tracking url
+It is possible to add a tracking URL for customers to monitor the delivery status.
+
+```xml
+// manifest.xml
+
+<?xml version="1.0" encoding="UTF-8" ?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/Manifest/Schema/manifest-2.0.xsd">
+    
+    ...
+
+    <shipping-methods>
+
+        <shipping-method>
+            <identifier>NameOfYourFirstShippingMethod</identifier>
+            <name>First shipping method</name>
+            ...
+            <tracking-url>https://www.yourtrackingurl.com</tracking-url>
+            ...
+        </shipping-method>
+
+    </shipping-methods>
+</manifest>
+```
```

Comment:
### Position
Here, you can set the display order of the shipping methods in the checkout.
If the tag is omitted, the position of the shipping method is 1 by default.

```xml
// manifest.xml

<?xml version="1.0" encoding="UTF-8" ?>
<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/Manifest/Schema/manifest-2.0.xsd">

    ...

    <shipping-methods>

        <shipping-method>
            <identifier>NameOfYourFirstShippingMethod</identifier>
            <name>First shipping method</name>
            ...
            <position>2</position>
            ...
        </shipping-method>

    </shipping-methods>
</manifest>
```